packages/plugin-conversation/src/conversation.js

```diff
@@ -502,6 +502,35 @@ const Conversation = SparkPlugin.extend({
   },
 
   /**
+   * Sets the typing status of the current user in a conversation
+   *
+   * @param {Object} conversation
+   * @param {Object} options
+   * @param {boolean} options.typing
+   * @returns {Promise}
+   */
+  updateTypingStatus(conversation, options) {
+    let eventType;
```

Comment:
Add a check to confirm the conversation has an id. Consider attempting to fallback to url first:

```javascript
if (!conversation.id) {
  if (conversation.url) {
    conversation.id = conversation.url.split(`/`).pop();
  } 
  else {
    return Promise.reject(new Error(`conversation: could not identify conversation`));
  }
}
```